src/components/Layout/AppText.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from "react";
+import { TextProps, Text } from "react-native";
+import { fontSize, color } from "../../common/styles";
+
+export const AppText: FunctionComponent<TextProps> = ({

```

Comment:
With the addition of this, will need to change the existing usages of `Text` too. Given the large extent of such a change, I suggest to move all the styling stuff out to another PR.

Ultimately error tracking and font loader aren't related